src/python/qgspythonutilsimpl.cpp

```diff
@@ -91,8 +93,14 @@ def run_startup_script(script_path):
         )
 
     return script_executed
-)"""" ).arg( pythonPath() ) );
-  runString( QStringLiteral( "run_startup_script(pyqgstart)" ) );
+        """,
+        'inline string in qgspythonutilsimpl.cpp',
```

Comment:
How about:

```suggestion
        'QgsPythonUtilsImpl::run_startup_script',
```

That might make debugging a little clearer

Reply:
We can change this, but I think it still may be confusing.  The C++ class is mixed with Python function name. Someone may want to search for C++ method with such name and would find nothing.
What about: `QgsPythonUtilsImpl::checkSystemImports [run_startup_script]` with clear distinction between C++ and python names?

Reply:
sounds good to me!